nx_parallel/algorithms/centrality/tests/test_betweenness_centrality.py

```diff
@@ -4,29 +4,34 @@
 
 
 def test_betweenness_centrality_get_chunks():
-    def get_chunk(nodes):
-        num_chunks = nxp.cpu_count()
-        nodes_ebc = {i: 0 for i in nodes}
-        for i in ebc:
-            nodes_ebc[i[0]] += ebc[i]
-            nodes_ebc[i[1]] += ebc[i]
+    def get_chunk_function(ebc):
```

Comment:
I don't think there is any need for 2 functions here(i.e. `get_chunk_function` and `get_chunk`) and `ebc` doesn't need to be passed as it is already in the outer function's namespace. LMK what you think. Thanks!